prac_07/project_management.py

```diff
@@ -0,0 +1,107 @@
+"""
+CP1404 Prac 07 - project_management.py
+Estimated time: 60m
+Actual time:
```

Comment:
This was a rough practical week

Reply:
Great feedback, caught a few things I didn't think of. Most of the issues were due to me having rush the completion of the prac. Thanks for the feedback : )